yamale/validators/tests/test_validate.py

```diff
@@ -22,6 +38,135 @@ def test_string():
     assert not v.is_valid(1)
 
 
+def test_validator_pathname():
+    '''
+    Unit test the `Pathname` validator.
+    '''
+
+    v = val.Pathname()
+
+    # Directory guaranteed to exist. See Pathname._is_valid().
+    root_dirname = os.environ.get('HOMEDRIVE', 'C:') \
```

Comment:
> Line continuations are fragile...

Readability concerns I grok, but... _fragile_? I've never had a line continuation fall down on me, but have encountered this opinion in the Python community before. There's even a [rejected PEP](https://www.python.org/dev/peps/pep-3125) arguing for the removal of explicit line continuations from the language altogether. I'm not quite sure what drives this, but suspect that Python developers attempting to distance themselves from lower-level languages (e.g., C, `sh`) and competitors (e.g., Ruby) accounts for a bit of it.

_Anyway._ I'll happily do it, of course; no troublesome complaints from me. Merely curious! :octocat: